requirements.txt

```diff
@@ -22,6 +22,7 @@ numpy>=1.10.1
 pexpect>=4.0.1
 psutil>=4.2.0
 Pygments>=1.5
+parameterized
```

Comment:
What's the difference with the `parametrize` included in `pytest`?

Reply:
It also works for unittest based tests

Reply:
I did not reallise until after making the pr that I had this dependency in the tests. I can of course rebase on any other branch.